packages/jaeger-ui/src/utils/filter-spans.tsx

```diff
@@ -60,7 +60,7 @@ export default function filterSpans(textFilter: string, spans: Span[] | TNil) {
     isTextInFilters(includeFilters, span.operationName) ||
     isTextInFilters(includeFilters, span.process.serviceName) ||
     isTextInKeyValues(span.tags) ||
-    (span.logs !== null && span.logs.some(log => isTextInKeyValues(log.fields))) ||
+    (span.logs !== null && span.logs !== undefined && span.logs.some(log => isTextInKeyValues(log.fields))) ||
```

Comment:
Perhaps just use `Array.isArray` instead of null checks. It would be even better if the spans were to go through some normalization phase where these model inconsistencies are fixed once (e.g. `logs=[]`) instead of having these checks all over the place.

If you want to see this PR through to the end, then
  * make sure commits are signed (DCO check is failing)
  * add a unit test for this condition